src/leetCodeExecutor.ts

```diff
@@ -61,9 +67,11 @@ class LeetCodeExecutor implements Disposable {
             try { // Check plugin
                 await this.executeCommandEx(this.nodeExecutable, [await this.getLeetCodeBinaryPath(), "plugin", "-e", plugin]);
             } catch (error) { // Download plugin and activate
+                await this.removeOldCache();
                 await this.executeCommandEx(this.nodeExecutable, [await this.getLeetCodeBinaryPath(), "plugin", "-i", plugin]);
             }
         }
+        context.globalState.update(globalStateLeetcodeIsUserFresh, false);
```

Comment:
In the end of activate()  I set it to false, in the first it should be undefined so I use `(isUserFresh !== false)` logic to check.

Reply:
Then instead of calling it `isUserFresh`, it would be better to call it `HasInited` IMO. And set it to `true` at the end of `activate()`

The aim of that is to avoid the two falsy value `undefined` and `false` has two different meanings.

Reply:
Got it.